src/Translator.java

```diff
@@ -0,0 +1,18 @@
+import java.util.HashMap;
+
+public class Translator {
+    // Declare object //
+    HashMap<Integer, String> numericAlpha = new HashMap<>();
```

Comment:
Ik heb hier:  Map<Integer, String> numericAlpha = new HashMap<>();

Een map kan geen duplicates bevatten en een hashmap kan dat wel. hashmap houdt ook niet perse rekening met een insertion order.

Zie bijvoorbeeld deze info: https://www.ksolves.com/blog/java/difference-between-map-and-hashmap-in-java